### PR TITLE
Add WP-CLI landing page

### LINKED ISSUE #159: Layout: WP CLI Landing
## Screenshot
[Figma Link](https://www.figma.com/file/2WxlJFzMJvqPfZL1EkAOVp/Developer-Pages?node-id=1160%3A10487&t=pS0XoTexhVvCl9Ft-0)

image

### PR DESCRIPTION
**Pull request by adamwoodnz**
Closes #159 

## Screenshots
| Mobile | Table | Desktop |
| --- | --- | --- |
| localhost_8888_cli_commands_(iPhone 12 Pro) (1) | localhost_8888_cli_commands_ (1) | localhost_8888_cli_commands_ (2) |

### CODE REVIEW

**Comment by StevenDufresne** [comment]
How is this going? Have you had any time to work on this?

**Comment by adamwoodnz** [comment]
> How is this going? Have you had any time to work on this?

I have not sorry, About pages have ballooned. This just needs a code table block added at the bottom with the commands data rendered.

**Comment by StevenDufresne** [comment]
Alright, I'll take a look at this today.

**Comment by adamwoodnz** [comment]
> Alright, I'll take a look at this today.

Sorry I haven't been able to help out more.

The URL and template name don't really seem to match as I'd expect, but for me the template loads at http://localhost:8888/cli/commands/

**Comment by StevenDufresne** [comment]
I've updated the issue description with some screenshots.

I have added a dynamic block, `wporg/cli-command-table`, to render the table. 

I considered an approach that uses a query block to build out the table using a post template but that seemed like an improvement that could be made in the future seeing that we already have all the code in place to just render the dynamic block and currently have no granular table components that could be used like that. 

Do you mind taking a look to see if it makes sense to you?

File: source/wp-content/themes/wporg-developer-2023/src/cli-command-table/block.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	$posts = get_posts(
+		array(
+			'post_type'      => 'command',
+			'order'          => 'ASC',
+			'post_parent'    => 0,
+			'nopaging'       => true,
+		)
+	);
+
+	if ( is_wp_error( $posts ) || empty( $posts ) ) {
```
Comment (StevenDufresne) [inline source/wp-content/themes/wporg-developer-2023/src/cli-command-table/block.php:37]: I think this is right but I can't find any documentation as to whether `is_wp_error( $posts )` should be checked. I've just seen it around the code base in similar scenarios.

**Comment by adamwoodnz** [comment]
Looking good! Minor but the column widths on the tables are more like 40/60 in the designs.

**Comment by adamwoodnz** [comment]
Line height on the description column text is a little tight, Figma has 30px or 1.88

Screen Shot 2023-02-10 at 4 52 41 PM

**Review by adamwoodnz — COMMENTED** [review]
I can't approve, but LGTM 🎉 

One minor comment, thanks for finishing this off.

**Comment by StevenDufresne** [comment]
> Looking good! Minor but the column widths on the tables are more like 40/60 in the designs.

Good call. Updated in https://github.com/WordPress/wporg-developer/pull/182/commits/c9baebeccbaa984e28910a718802749d30861e08.

**Pull request merged**